src/App.tsx

```diff
@@ -10,31 +14,29 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setselectedTabId] = useState(tabs[0]);
+
+  const tabSelection = (tab: Tab) => {
+    setselectedTabId(tab);
+  };
+
   return (
     <div className="section">
       <h1 className="title">
-        Selected tab is Tab 1
+        {`Selected tab is ${selectedTabId.title}`}
       </h1>
 
       <div data-cy="TabsComponent">
         <div className="tabs is-boxed">
-          <ul>
-            <li className="is-active" data-cy="Tab">
-              <a href="#tab-1" data-cy="TabLink">Tab 1</a>
-            </li>
-
-            <li data-cy="Tab">
-              <a href="#tab-2" data-cy="TabLink">Tab 2</a>
-            </li>
-
-            <li data-cy="Tab">
-              <a href="#tab-3" data-cy="TabLink">Tab 3</a>
-            </li>
-          </ul>
+          <Tabs
+            tabs={tabs}
+            selectedTabId={selectedTabId.id}
+            onTabSelected={tabSelection}
+          />
         </div>
 
         <div className="block" data-cy="TabContent">
-          Some text 1
+          {selectedTabId.content}
```

Comment:
The content should be inside the Tabs component. You should find a tab content according to the tab.id and show found tab or tabs[0] to pass this test
<img width="589" alt="image" src="https://user-images.githubusercontent.com/99403198/220078373-63c85039-5e2b-4805-8ffa-26d877f8230c.png">